index.html

```diff
@@ -25,12 +25,12 @@
         <ul class="collapsible" data-collapsible="expandable" id="searchDetails">
           <li>
             <div class="collapsible-header">
-              <i class="material-icons">place</i>Luby's</div>
+              <i class="material-icons">home</i>Add Address Below</div>
             <div class="collapsible-body">
-              <span>Address info, Yelp rating, Yelp price</span>
+              <span></span>
             </div>
           </li>
-          <li>
+          <!-- <li>
```

Comment:
I think we might as well cut this out if addresses are loading dynamically now

Reply:
The only reason I left one in is for aesthetics. Otherwise, the right side looks a little blank.

Reply:
Oh, unless you meant the ones that are commented out. I can delete those.